indico/modules/rb_new/controllers/management/contributions.py

```diff
@@ -0,0 +1,40 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2019 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
+
+from flask import jsonify, session
+
+from indico.modules.events.contributions import Contribution
+from indico.modules.events.management.controllers import RHManageEventBase
+
+
+class RHListOtherContributions(RHManageEventBase):
+    """AJAX endpoint that lists all contributions in the event (dict representation)."""
```

Comment:
not sure what the "(dict representation)" means here, you return a list ;) (and i think the docstring is more than clear enough without that)

Reply:
Copy paste from https://github.com/indico/indico/blob/master/indico/modules/events/abstracts/controllers/reviewing.py#L46 :confused:

Reply:
OK weird. Anyway, then the "other" makes more sense too since for the abstract one it only shows abstracts that are not the one the user currently views